schemas/cloud.yml

```diff
@@ -80,6 +80,16 @@
 
         Examples: AWS account name, Google Cloud ORG display name.
 
+    - name: platform
+      level: extended
+      type: keyword
+      example: lambda
+      short: The cloud platform name.
+      description: >
+        The cloud platform name.
```

Comment:
The PR's description contains extra context that I think would be useful to help users understand this field. Consider adding some of that info into the description.

Reply:
done